app/internal/app/reduced_fee_store.go

```diff
@@ -0,0 +1,49 @@
+package app
+
+import (
+	"context"
+	"time"
+
+	"github.com/ministryofjustice/opg-modernising-lpa/app/internal/page"
+)
+
+type reducedFeeStore struct {
+	dynamoClient DynamoClient
+	now          func() time.Time
+}
+
+type reducedFee struct {
+	PK           string
+	SK           string
+	PaymentID    string
+	LpaUID       string
+	FeeType      string
+	Amount       int
+	UpdatedAt    time.Time
+	EvidenceKeys []string
+}
+
+func (r *reducedFeeStore) Create(ctx context.Context, lpa *page.Lpa) error {
+	reducedFee := &reducedFee{
+		PK:        "LPAUID#" + lpa.UID,
+		SK:        "#PAYMENT#" + lpa.PaymentDetails.PaymentId,
```

Comment:
The table was set up with just a primary key but as we're relying on the underlying condition set to check for no duplicate PK+SKs it would limit multiple payments against an LPA UID not having an SK. I'd started to extend the client to make that condition toggleable but opted for this instead. I do agree though we aren't actively using the SK for anything at the moment so I could just use the UNIX time for now until a need arises.